02-electric/electric-section-one/electric-section-one.html

```diff
@@ -13,69 +13,109 @@
   </head>
   <body>
     <section class="electric-section-one-container">
+
       <section class="electric-section-one-subcontainer">
-        <h1 class="electric-section-one-title">Lets Talk Laundry</h1>
-        <h2 class="electric-section-one-subtitle">Laundry and Electricity</h2>
-
-        <div class="electric-section-one-part-one-img-container">
-          <img
-            class="electric-section-one-part-one-img"
-            src="../../00-overall/05-images/placeholder.svg"
-            alt="image section two"
-          />
-        </div>
-        <h2 class="electric-section-one-part-one-title">
-          Washing clothes at a lower temperature could save you up to £40 a year
-        </h2>
-
-        <p class="electric-section-one-part-one-paragraph">
-          Modern washing machines can clean clothes effectively at lower
-          temperatures. Changing from 40°C to 30°C means you could get 3 cycles
-          instead of 2 using the same amount of energy, depending on your
-          washing machine.
-        </p>
-        <div class="electric-section-one-part-two-img-container">
-          <img
-            class="electric-section-one-part-two-img"
-            src="../../00-overall/05-images/placeholder.svg"
-            alt="image section two"
-          />
+
+        <h1 class="electric-section-one-title">
+          Laundry and Electricity
+        </h1>
+
+        <div class="electric-section-one-part-one-container">
+
+          <div class="electric-section-one-part-one-image-container">
+            <img
+              class="electric-section-one-part-one-image"
+              src="../../00-overall/05-images/placeholder.svg"
+              alt="image section two"
+            />
+          </div>
+
+          <div class="electric-section-one-part-one-copy-container">
+
+            <h2 class="electric-section-one-part-one-subtitle">
+              Washing clothes at a 
+              <span class="bold">lower temperature </span>
+              could save you up to 
+              <span class="bold">£40 a year</span>
+            </h2>
+
+            <div class="electric-section-one-part-one-paragraph-container">
+              <p class="electric-section-one-part-one-paragraph">
+                Modern washing machines can clean clothes effectively at lower
```

Comment:
please make sure all text in paragraphs is a single line that is well formatted
it is not good practice to separate the paragaph up like this
if you need to see it in multiple lines in vscode you can use alt+z to toggle an option

Reply:
done